src/app/filter/filter.component.html

```diff
@@ -0,0 +1,41 @@
+<form class="filter__form" (ngSubmit)="onSubmit()">
+  <div class="filter__wrapper">
+    <label for="currentPage">Page:</label>
+    <input
+      class="filter__input filter__input--small"
+      type="number"
+      min="1"
+      placeholder="Page index"
+      [(ngModel)]="currentPage"
```

Comment:
that's ok for now, but usually it is better to use ReativeForms and avoid ngModel, if you can you can do a little refactoring and do not use ngmodel